src/mento.ts

```diff
@@ -82,7 +82,8 @@ export class Mento {
    */
   static createWithBrokerAddress(
     brokerAddr: Address,
-    signerOrProvider: Signer | providers.Provider
+    signerOrProvider: Signer | providers.Provider,
+    exchanges?: Exchange[]
```

Comment:
what do you think of renaming `createWithBrokerAddress` to something like `createWithParams` and keeping the `create` method as is? Seems like if a consumer is willing to provide the exchanges config they will know the brokerAddress as well, and we can keep the full "discovery" in the `create` method.

also: could you also update the docs in the method? 🙏